### ISSUE TITLE
Run production error

### ISSUE THREAD

**Issue by JohnnyLe** [wealthbot-io/wealthbot#73]
Hi all,

I'm setting up to run latest version in EC2 LAMP stack, I follow document https://wealthbot.readme.io/docs/production-install

But in this step 

```
app/console doctrine:schema:create
app/console doctrine:fixtures:load
```

It have bellow error

![image](https://cloud.githubusercontent.com/assets/1794584/15048679/b7ba4156-1314-11e6-8376-801a44edc5ff.png)

Does anyone have the same issue?

Best regards
Johnny

**Comment by JohnnyLe** [comment]
I have resolved this issue above by enable "extension=apcu.so" in php.ini

But go to next step "app/console doctrine:fixtures:load"  is failed authentication with MongoDB

![image](https://cloud.githubusercontent.com/assets/1794584/15052908/317e1822-132a-11e6-9b15-913b7dda3b5c.png)

But I test the account from command line, it can login normally  

![image](https://cloud.githubusercontent.com/assets/1794584/15052976/8415855c-132a-11e6-874b-c8b46815cf71.png)

Does anyone have the same issue?

**Comment by gkobilansky** [comment]
Hi @JohnnyLe. Did you set the MongoDB password here - https://github.com/wealthbot-io/wealthbot/blob/master/app/config/parameters.yml.dist ?

**Comment by JohnnyLe** [comment]
Thank you @gkobilansky for the response.

Oh, I just set in /app/config/parameters.yml let me try again.

Best regards
Johnny

**Comment by JohnnyLe** [comment]
Hi @gkobilansky,

I tried again but the result is the same, it's quite strange. Bellow are my config database in both file ```parameters.yml``` and ```parameters.yml.dist```. Did you see any wrong or ny other file to config more. All database account have been verified can login by linux command line.

1) In file  /app/config/parameters.yml
```
parameters:
    database_driver:   pdo_mysql
    database_host:     127.0.0.1
    database_port:     3306
    database_name:     wealthbot
    database_user:     root
    database_password: 'pss1@#'

    mongodb_host: 127.0.0.1
    mongodb_database: wealthbot
    mongodb_user: wealthbot
    mongodb_password: 'pss1@#'
```

2) In file /app/config/parameters.yml.dist
```
parameters:
    database_driver:   pdo_mysql
    database_host:     127.0.0.1
    database_port:     3306
    database_name:     wealthbot
    database_user:     root
    database_password: 'pss1@#'

    mongodb_host: 127.0.0.1
    mongodb_database: wealthbot
    mongodb_user: wealthbot
    mongodb_password: 'pss1@#'
```

Best regards
Johnny

**Comment by JohnnyLe** [comment]
Hi @gkobilansky,

Also do we have any tags release for stable build? Because i'm trying with latest build from master branch.

Best regards
Johnny

**Comment by JohnnyLe** [comment]
I have figured out the problem of Mongodb authentication failed.
It was caused by a problem/bug in the PHP mongo driver version 1.4
After upgraded the driver to version 1.6 it have fixed. Run command line
```
pecl upgrade mongo
```
You should note it into the release note as well.
The same issue http://stackoverflow.com/questions/29792674/cant-authenticate-on-mongodb-with-php

Now everything run without error 
```
app/console doctrine:database:create
app/console doctrine:schema:create
app/console doctrine:fixtures:load
```

But after run ```app/console doctrine:fixtures:load``` completed. I go to home page it shows error

>Fatal error: Call to undefined function Doctrine\Common\Cache\apc_fetch() in /apps/wealthbot/vendor/doctrine/cache/lib/Doctrine/Common/Cache/ApcCache.php on line 41

Really strange again, the php-apc have installed
```
sudo pecl install apcu
```

All php.ini have enable 
/etc/php/php.ini
/apps/wealthbot/vendor/gedmo/doctrine-extensions/tests/travis/php.ini
/apps/wealthbot/vendor/doctrine/cache/tests/travis/php.ini
```
extension=apcu.so
apc.enable_cli=1
```

Did you get the same error in the last?

Best regards
Johnny

**Comment by JohnnyLe** [comment]
Hi @gkobilansky 

Sorry it seem the error message ```Call to undefined function Doctrine\Common\Cache\apc_fetch()``` is caused by chrome cache. Now clear all cache the error message at home page is

![image](https://cloud.githubusercontent.com/assets/1794584/15092875/d0f047d2-14a0-11e6-8a1b-f6cf57d453ba.png)

Best regards
Johnny

**Comment by JohnnyLe** [comment]
Hi,

Finally I have figure out the problem. It is the same issue http://stackoverflow.com/questions/36493327/symfony2-install-failing-error-1866-attribute-path-is-not-allowed-fileloa

So just post here if anyone can avoid

Correct answer 
>Symfony 2.1 (which by the way dropped support a long time ago) did not have the ```path``` attribute for routes. You had to use the ```pattern``` attribute then.


Currently you still use ```path``` attribute in /wealthbot/vendor/friendsofsymfony/jsrouting-bundle/Resources/config/routing/routing.xml

Thanks
Johnny

**Comment by gkobilansky** [comment]
Hi @JohnnyLe,

Thanks for all the hard work over the weekend! Apologies for my delay in responding.

I will update install instructions regarding PHP mongo driver version 1.4 vs. PHP mongo driver version 1.6.

You should know that we are in the process of upgrading to Symfony 2.8, (expected release in a week or so) so the path/pattern issue will be resolved as part of that release.

I will close this issue because it was originally opened for MongoDB authentication.

Best,
Gene

**Comment by pandiyanmkm** [comment]
hi all.

can anyone please give a proper guidelines to install a wealthbot in localhost windows (developement) and godaddy (production) server.

i have trying to install using this guidelines ,

https://wealthbot.readme.io/docs/getting-started

but not able to install properly in my local.

kindly advice me
